src/DeviceInterface.js

```diff
@@ -45,6 +45,7 @@ function createDevice () {
         }
         return new AppleDeviceInterface(globalConfig, deviceApi, settings)
     }
+    globalConfig.isExtension = true
```

Comment:
I ended up setting it here. All other similar properties are in the `globalConfig` so it makes sense, but extensions don't inject strings on the fly so we can do it like this here. Any egregious downside?

FYI, using `instanceof ExtensionInterface` couldn't be used because imports were not failing.

Reply:
I'd say for now this is fine given how we're determining that we're in the extension - as you can imagine there are other ways to be more deterministic, but given what we have currently I don't think this is an issue at all :)